styles/main.css

```diff
@@ -0,0 +1,230 @@
+:root {
+  /* defining varibles */
+  --primary-color: #ff3b30; /*#fca5a5*/
+  --secondary-color: #fff6ea;
+  --action: #3b82f6;
+  --grey: #9a9a9a;
+  --black: #2b2b2b;
+  --white: #ffffff;
+  --lynx-white: #f7f7f7;
+  --violet: #2348c7;
+  --lynx-gray: #c5c5c5;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  background-color: var(--white);
+  color: var(--black);
+  box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+}
+
+.small-text {
+  font-size: 0.6rem;
+  font-weight: 500;
+  color: var(--black);
+}
+
+.nav {
+  text-decoration: none;
+  font-size: 1.4rem;
+  font-weight: 500;
+  color: var(--black);
+}
+
+.nav:hover {
+  color: var(--primary-color);
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  gap: 2rem;
+}
+
+.section {
+  /* margin: 6rem; */
+  margin-top: 6rem;
+  color: var(--black);
+}
+
+.section-main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  border: #ef4444;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  font-size: 3rem;
+}
+
+@media screen and (max-width: 768px) {
+  .container {
+    font-size: 2rem;
+  }
+}
+
+@media screen and (max-width: 640px) {
+  .container {
+    font-size: 1.6rem;
+  }
+}
+
+@media screen and (max-width: 480px) {
+  .container {
+    font-size: 1.2rem;
+  }
+}
+
+.about {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  font-size: 1.5rem;
+  font-weight: 500;
+  margin: 3rem 5rem 3rem 5rem;
+}
+
+.btn-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 5rem;
+}
+
+.btn {
+  text-decoration: none;
+  background: transparent;
+  font-size: 1.2rem;
+  font-weight: 500;
+  cursor: pointer;
+  padding: 1rem 2rem;
+  border-radius: 10px;
+}
+
+.btn-primary {
+  border: 1px solid var(--primary-color);
```

Comment:
Keep all media queries in end...and  group/write similar styles together. It will help you to easy to kind files

Reply:
Done!! Keep all the queries in the last of the file.